org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/hotspots/HotspotsView.java

```diff
@@ -0,0 +1,292 @@
+    emptyMsg.setText("You can open a Security Hotspot from SonarQube. <a>More infos</a>");
+    GridData gd = new GridData(SWT.LEFT, SWT.FILL, true, false);
+    emptyMsg.setLayoutData(gd);
+    emptyMsg.setBackground(emptyMsg.getDisplay().getSystemColor(SWT.COLOR_LIST_BACKGROUND));
+    emptyMsg.addSelectionListener(new SelectionAdapter() {
+      @Override
+      public void widgetSelected(SelectionEvent e) {
+        try {
+          PlatformUI.getWorkbench().getBrowserSupport().getExternalBrowser().openURL(new URL("https://github.com/SonarSource/sonarlint-eclipse/wiki/Security-Hotspots"));
+        } catch (PartInitException | MalformedURLException e1) {
+          // ignore
+        }
+      }
+    });
+    return form;
+  }
+
+  private Control createHotspotsPage(FormToolkit kit) {
+    Form form = kit.createForm(book);
+    Composite body = form.getBody();
+    GridLayout layout = new GridLayout();
+    body.setLayout(layout);
+
+    splitter = new SashForm(body, SWT.NONE);
+    GridData gd = new GridData(SWT.FILL, SWT.FILL, true, true);
+    splitter.setLayoutData(gd);
+    splitter.setOrientation(SWT.HORIZONTAL);
+
+    createHotspotTable();
+
+    final TabFolder tabFolder = new TabFolder(splitter, SWT.NONE);
+
+    TabItem riskDescriptionTab = new TabItem(tabFolder, SWT.NONE);
+    riskDescriptionTab.setText("What's the risk?");
+    riskDescriptionTab.setToolTipText("Risk decription");
+    riskDescriptionTab.setControl(createRiskDescriptionControl(tabFolder));
+
+    TabItem vulnerabilityDescriptionTab = new TabItem(tabFolder, SWT.NONE);
+    vulnerabilityDescriptionTab.setText("Are you at risk?");
+    vulnerabilityDescriptionTab.setToolTipText("Vulnerability decription");
+    vulnerabilityDescriptionTab.setControl(createVulnerabilityDescriptionControl(tabFolder));
+
+    TabItem fixRecommendationsTab = new TabItem(tabFolder, SWT.NONE);
+    fixRecommendationsTab.setText("How can you fix it?");
+    fixRecommendationsTab.setToolTipText("Recommendations");
+    fixRecommendationsTab.setControl(createFixRecommendationsControl(tabFolder));
+
+    return form;
+  }
+
+  private Control createRiskDescriptionControl(Composite parent) {
+    riskDescriptionBrowser = new SonarLintWebView(parent, true) {
+
+      @Override
+      protected String body() {
+        ServerHotspot hotspot = (ServerHotspot) hotspotViewer.getStructuredSelection().getFirstElement();
+        if (hotspot == null) {
+          return NO_SECURITY_HOTSPOTS_SELECTED;
+        }
+        return hotspot.rule.riskDescription;
+      }
+    };
+    return riskDescriptionBrowser;
+  }
+
+  private Control createVulnerabilityDescriptionControl(Composite parent) {
+    vulnerabilityDescriptionBrowser = new SonarLintWebView(parent, true) {
+
+      @Override
+      protected String body() {
+        ServerHotspot hotspot = (ServerHotspot) hotspotViewer.getStructuredSelection().getFirstElement();
+        if (hotspot == null) {
+          return NO_SECURITY_HOTSPOTS_SELECTED;
```

Comment:
Could this happen ? Couldn't we hide the tabs if no hotspot is selected ?
In IJ, we show the tabs even if the selection is not active, we show them as soon as a hotspot has been opened.

Reply:
Today it is possible to unselect the hotspot. We could prevent that maybe. We'll see later.